new/LIT7072QeneMamhera.xml

```diff
@@ -5,9 +5,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">">መምህረ፡ ዝንቱ፡ ዓለም፡ አውስጣቴዎስ፡ ደብረ፡ አርማንያ፡ ወልደከ፡ በጽሐ፡ ለዘምሮ፤</title>
-                <title xml:lang="gez" corresp="#t1" type="normalized">Mamǝhǝra zǝntu ʿālam, ʾAwǝsṭātewos, dabra ʾArmānyā waldaka baṣḥa la-zammǝro… (Qǝne of unknown type)</title>
-                <title xml:lang="en" corresp="#t1">The teacher of this world, ʾEwosṭātewos, came to the mountain of Armenia to praise Your Son…</title>
+                <title xml:lang="gez" xml:id="t1">መምህረ፡ ዝንቱ፡ ዓለም፡ <choice><sic>አ</sic><corr resp="JK">ኤ</corr></choice>ዎስጣቴዎስ፡ ደብረ፡ አርማንያ፡ ወልደከ፡ በጽሐ፡ ለዘምሮ፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Mamhǝra zǝntu ʿālam, ʾEwosṭātewos, dabra ʾArmānyā waldaka baṣḥa la-zammǝro (Qǝne of unknown type)</title>
```

Comment:
I meant this: Mamhǝra zǝntu ʿālam ʾEwosṭātewos dabra ʾArmānyā waldaka baṣḥa la-zammǝro, ...